cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+openapi: 3.1.0
+info:
+  version: 0.7.0
```

Comment:
Dear Reviewer

thank you for your valuable feedback. Please provide your feedback on the corresponding point in this file and include your feedback in the follow structure:

- Title:
- Question/Feedback:
- Suggestion for improvement:

Please start with describing your Feedback or asking your Question regarding a specific component in the spec. For overall questions or remarks please use the comment function for the whole file. Make sure your feedback is not collected already, if so, please like the feedback or add further issues or clarifications as answer to the comment.

It might be very helpful if you suggest improvements for the core team to elaborate your feedback. Thus we ask you to share all your relevant thoughts on this topic and feel free to share any solutions to your issue. This will improve further discussion and allow people to refactor your suggestions. The core team will come back to your feedback as needed.